docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+services:
+  bot-librarian:
+    build: .
+    container_name: bot-librarian
+    image: bot-librarian
+    command: node ./bin/bot-librarian.js
+    ports:
+      - "${BOT_EXTERNAL_PORT-:5000}:${BOT_INTERNAL_PORT:-5000}"
+    environment:
+      - PORT=${PORT:-5000}
```

Comment:
Why `5000` port? On our current instance of Digital Ocean it was set to different port in Environment Variables